client/my-sites/plan-features-2023-grid/components/actions.tsx

```diff
@@ -360,15 +356,16 @@ const PlanFeaturesActionsButton: React.FC< PlanFeaturesActionsButtonProps > = (
 	const translate = useTranslate();
 	const isEnglishLocale = useIsEnglishLocale();
 	const { gridPlansIndex } = usePlansGridContext();
+	const {
+		planTitle,
+		current,
+		pricing: { currencyCode, originalPrice, discountedPrice },
+	} = gridPlansIndex[ planSlug ];
```

Comment:
Nit. This looks like a repetitive pattern, maybe it can be moved into it's own hook. 

```javascript
     const gridPlan = useContextGridPlan(planSlug);
```

Reply:
Not a bad idea. I guess we'll see what becomes of this context and whether we'll design more around it. Still not sure if the data-store, that we use already, isn't better fitted for these. The more we move into context I think the more we risk forcing unnecessary rerenders 🤷‍♂️